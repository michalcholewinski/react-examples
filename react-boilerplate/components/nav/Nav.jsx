import Link from 'next/link';
import React from 'react';

export const Nav = () => (
  <nav>
    <ul>
      <li>
        <Link prefetch href="/">
          <a>Home</a>
        </Link>
      </li>
      <li>
        <Link prefetch href="/anotherPage">
          <a>Another Page</a>
        </Link>
      </li>
    </ul>
  </nav>
);
