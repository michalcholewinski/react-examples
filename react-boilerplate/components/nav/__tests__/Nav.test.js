import { shallow } from 'enzyme';
import React from 'react';

import { Nav } from '../Nav';

describe('Nav', () => {
  it('should match snapshot', () => {
    const component = shallow(<Nav />);

    expect(component)
      .toMatchSnapshot();
  });
});
