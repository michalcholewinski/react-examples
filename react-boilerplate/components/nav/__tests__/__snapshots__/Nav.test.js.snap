// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match snapshot 1`] = `
<nav>
  <ul>
    <li>
      <Link
        href="/"
        prefetch={true}
      >
        <a>
          Home
        </a>
      </Link>
    </li>
    <li>
      <Link
        href="/anotherPage"
        prefetch={true}
      >
        <a>
          Another Page
        </a>
      </Link>
    </li>
  </ul>
</nav>
`;
